docs/26_export_manual.md

```diff
@@ -0,0 +1,269 @@
+% Data Export
+% Colorado Risk Limiting Audit Tool 
+% 2017
+
+Separate from the RLA central server and client server programs that support the Department of State and the Counties in carrying out the Risk Limiting Audit, there is a command, called `rla_export`, allowing export of data from the central server and the underlying database. 
+
+The command is part of a python package, whose technical description can be found in a `README.rst` file in the python site-packages directory tree wherever the package is installed. The most current online version (which may or may not match the version you have installed) is available 
+in the [public code repository](https://github.com/FreeAndFair/ColoradoRLA/tree/auditcenter/server/eclipse-project/script/rla_export).
+
+The `README.rst` file gives instructions for installing the python package, and describes various run-time options. For a catalog of the exports produced by the command, see below.
```

Comment:
I think we should include an HTML or pdf version of the README.rst, and leave out these references to the source, and installing etc. Just describe the exported fields, and include the actual Python package as part of the install zip.